target_chains/sui/contracts/sources/pyth.move

```diff
@@ -348,20 +445,23 @@ module pyth::pyth_tests{
 
     use pyth::state::{State as PythState};
     use pyth::setup::{Self};
-    //use pyth::price_identifier::{Self};
-    use pyth::price_info::{PriceInfo, PriceInfoObject};//, PriceInfo, PriceInfoObject};
-    //use pyth::price_feed::{Self};
+    use pyth::price_info::{Self, PriceInfo, PriceInfoObject};//, PriceInfo, PriceInfoObject};
     use pyth::data_source::{Self, DataSource};
     //use pyth::i64::{Self};
     //use pyth::price::{Self};
-    use pyth::pyth::{Self, create_price_infos_hot_potato, update_single_price_feed};
-    use pyth::hot_potato_vector::{Self};
+    use pyth::pyth::{Self, create_authenticated_price_infos_using_batch_price_attestation, update_single_price_feed};
+    use pyth::authenticated_vector::{Self};
+    use pyth::price_identifier::{Self};
+    use pyth::price_feed::{Self};
+    use pyth::accumulator::{Self};
+    use pyth::deserialize::{Self};
 
     use wormhole::setup::{Self as wormhole_setup, DeployerCap};
     use wormhole::external_address::{Self};
     use wormhole::bytes32::{Self};
     use wormhole::state::{State as WormState};
     use wormhole::vaa::{Self, VAA};
+    use wormhole::cursor::{Self};
```

Comment:
does the VAA below need to be declared test_only when the whole module is already test_only ?

Reply:
yeah you're right, the `#[test_only]`s is an artifact of refactoring the tests (moving some from from `pyth_accumulator.move` to `pyth.move`). In the former file, the tests were not located in a `test_only` module, but in the latter they are.